### PR TITLE
chore: fix tool.lint.ruff in pyproject.toml

### PR DESCRIPTION
**Pull request by nobu007**
Fix setting in pyproject.toml.
- tool.ruff -> tool.lint.ruff 
- ruff version = 0.4.1.

### This PR resolve Warnings

result(before).

```bash
langchain-google$ ruff formatruff format
warning: The top-level linter settings are deprecated in favour of their counterparts in the `lint` section. Please update the following options in `langchain-google/libs/genai/pyproject.toml`:
  - 'select' -> 'lint.select'
warning: The top-level linter settings are deprecated in favour of their counterparts in the `lint` section. Please update the following options in `langchain-google/libs/community/pyproject.toml`:
  - 'select' -> 'lint.select'
warning: The top-level linter settings are deprecated in favour of their counterparts in the `lint` section. Please update the following options in `langchain-google/libs/vertexai/pyproject.toml`:
  - 'select' -> 'lint.select'
4 files reformatted, 122 files left unchanged
```

result(before).

```
langchain-google$ ruff format
52 files reformatted, 74 files left unchanged
```

### CODE REVIEW

File: libs/community/pyproject.toml [hunk 1]
```diff
@@ -101,7 +101,7 @@ types-requests = "^2.31.0.10"
 types-pillow = "^10.1.0.2"
 types-google-cloud-ndb = "^2.2.0.1"
 
-[tool.ruff]
+[tool.lint.ruff]
```
Comment (lkuligin) [inline libs/community/pyproject.toml:104]: should it be `tool.ruff.lint` too, or am I missing anything?

Comment (nobu007) [reply libs/community/pyproject.toml:108]: Sorry, I updated it.

**Review by lkuligin — APPROVED** [review]

**Pull request merged**